eZ/Publish/Core/Persistence/Cache/UrlAliasHandler.php

```diff
@@ -54,9 +53,7 @@ public function publishUrlAliasForLocation(
             ]
         );
 
-        $urlAliasHandler = $this->persistenceHandler->urlAliasHandler();
-
-        $urlAliasHandler->publishUrlAliasForLocation(
+        $urlAliasIdentity = $this->persistenceHandler->urlAliasHandler()->publishUrlAliasForLocation(
```

Comment:
I'd rather we do not combine the "acquire service" and "perform publishing" operations into one line, according a rule one statement should equal to one operation.

Reply:
Ok, fixed via 941c3128e5.